util/reggen/gen_rtl.py

```diff
@@ -46,6 +46,7 @@ def parse_field(obj, reg, nfields):
     f.hwre = obj["genhwre"]
     f.hwext = reg.hwext
     f.tags = obj["tags"]
+    f.shadowed = reg.shadowed
```

Comment:
Just a quick question: are we supporting shadowed register field as well?

Reply:
I would assume either a register is shadowed or not, not necessarily by field. But I'll have to read further to see if a shadowed register can have fields...

Reply:
yeah, looks like you can have a shadowed register with fields, but you can't have a register with some shadowed fields and other non-shadowed fields.

Reply:
Got it, thanks for the explanation.